text/0000-deprecate-ember-logger.md

```diff
@@ -0,0 +1,265 @@
+for Ember.Logging. The only anomaly remaining in Javascript engine console 
+support is that versions of Node earlier than Node 9 do not support 
+console.debug, so console.log will need to be used as the replacement for 
+Logging.debug, rather than console.debug, unless the minimum Node support 
+level is set to Node 9.
+
+The following high-impact packages (9 or 10 or a * on EmberObserver) are 
+calling Ember.Logging and should probably be given an early heads-up to 
+adjust their code to use console in advance of the release containing the 
+deprecation to limit the level of pain that their users experience. In 
+the order of their reliance on Ember.Logging:
+
+* ember-concurrency (15)
+* ember-cli-deprecation-workflow (9)
+* ember-stripe-service (9)
+* semantic-ui-ember (7)
+* ember-resolver (6)
+* ember-cli-page-object (4) 
+* ember-cli-sentry (3)
+* ember-islands (3)
+* ember-states (3)
+* ember-cli-pagination (2)
+* ember-cli-clipboard (1)
+* ember-cli-fastboot (1)
+* ember-elsewhere (1)
+* ember-i18n (1)
+* ember-simple-auth-token (1)
+* ember-svg-jar (1)
+* liquid-fire (1)
+
+### Timeline
+
+1. Remove all uses of Ember.Logging from other Ember and ember-data code.
+This is particularly important for any methods involved in reporting 
+deprecation. [Date? This could start now.]
+2. Write and test codemod to change code to use console and make it available. [Any time]
+3. Inform users that Ember.Logging will be deprecated and in what release. [When we know schedule]
+4. Call deprecation APIs from all Logging methods in @ember-console/index.js [Once codemod is out there]
+5. In documentation, offer suggestions for babel plugins available to suppress [Before we release]
+console calls in production builds with any special instructions for how to 
+use them in ember-cli broccoli builds.
+6. In due season, the deprecation will appear in a release. [When?]
+
+## How we teach this
+
+### Official code bases and documentation
+
+We do not currently actively teach the use of Ember.Logger. We will need to 
+remove any passing references to Ember.Logger from the Ember guides 
+and from the Super Rentals tutorial, if any such references exist. 
+
+Once it is gone from the code, we need to verify it no longer appears in the 
+API listings.
+
+Likewise, we will need to remove the use of Ember.Logger from ember, ember-data, 
+and the website.
+
+## Drawbacks
+
+191 projects in Ember Inspector are using it. It has been there and documented 
+for a long time. So this deprecation will cause API churn on a lot of projects.
+Hopefully providing a codemod to replace Ember.Logger calls with the 
+corresponding console calls will make this transition relatively painless.
+
+However, those using Logging.debug as something different from Logging.log
+may beg to differ, even though under the covers Logging.debug only calls 
+console.debug if it exists, calling console.log otherwise.
+
+## Alternatives
```

Comment:
Can you see some reasoning for not wanting to pursue the alternatives? It could help to preemptively address suggestions once the discussion starts.